splice_machine/src/test/java/com/splicemachine/triggers/Trigger_Referencing_Clause_IT.java

```diff
@@ -887,6 +887,7 @@ public void testNestedTriggers() throws Exception {
 
     // Cause trigger rows to go over 1000000, causing a
     // rollback and switch to spark execution.
+    @Ignore
```

Comment:
Should this test stay ignored? Is there a problem with test?

Reply:
the whole test class is still ignored. there's a couple of jiras:
- https://splicemachine.atlassian.net/browse/DB-9614
- https://splicemachine.atlassian.net/browse/DB-10539
- https://splicemachine.atlassian.net/browse/DB-10579

we're not fixing any of this here, right?

Reply:
We will need more work to fix these ITs

Reply:
To fix and re-enable these ITs, added https://splicemachine.atlassian.net/browse/DB-10616